app.rb

```diff
@@ -0,0 +1,29 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep'
+require './lib/time'
+
+class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    'Chitter page working!'
+  end
+
+  get '/peep' do
+    @peeps = Peep.all
+    # @time = Times.time
+    erb :'index'
+  end
+
+  post '/peeps' do
+    peep = params['peep']
+    connection = PG.connect(dbname: 'Peeps')
+    connection.exec("INSERT INTO Peeps (peep) VALUES('#{peep}')")
```

Comment:
Good use of a string interpolation to insert the params['peep'] into your table of peeps